run_bc.py

```diff
@@ -0,0 +1,113 @@
+            ``str`` type means config file path. \
+            ``Tuple[dict, dict]`` type means [user_config, create_cfg].
+        - seed (:obj:`int`): Random seed.
+        - env_setting (:obj:`Optional[List[Any]]`): A list with 3 elements: \
+            ``BaseEnv`` subclass, collector env config, and evaluator env config.
+        - model (:obj:`Optional[torch.nn.Module]`): Instance of torch.nn.Module.
+        - max_train_iter (:obj:`Optional[int]`): Maximum policy update iterations in training.
+        - max_env_step (:obj:`Optional[int]`): Maximum collected environment interaction steps.
+    Returns:
+        - policy (:obj:`Policy`): Converged policy.
+    """
+    if isinstance(input_cfg, str):
+        cfg, create_cfg = read_config(input_cfg)
+    else:
+        cfg, create_cfg = input_cfg
+    create_cfg.policy.type = create_cfg.policy.type + '_command'
+    env_fn = None if env_setting is None else env_setting[0]
+    cfg = compile_config(cfg, seed=seed, env=env_fn, auto=True, create_cfg=create_cfg, save_cfg=True)
+    # Create main components: env, policy
+    if env_setting is None:
+        env_fn, collector_env_cfg, evaluator_env_cfg = get_vec_env_setting(cfg.env)
+    else:
+        env_fn, collector_env_cfg, evaluator_env_cfg = env_setting
+    collector_env = create_env_manager(cfg.env.manager, [partial(env_fn, cfg=c) for c in collector_env_cfg])
+    evaluator_env = create_env_manager(cfg.env.manager, [partial(env_fn, cfg=c) for c in evaluator_env_cfg])
+    collector_env.seed(cfg.seed)
+    evaluator_env.seed(cfg.seed, dynamic_seed=False)
+    set_pkg_seed(cfg.seed, use_cuda=cfg.policy.cuda)
+    policy = create_policy(cfg.policy, model=model, enable_field=['learn', 'collect', 'eval', 'command'])
+    sd = torch.load(load_path, map_location='cpu')
+    policy.collect_mode.load_state_dict(sd)
+    return policy
+
+
+def run_go():
+    bipe_sac_config, bipe_sac_create_config = main_config, create_config
+    train_config = [deepcopy(bipe_sac_config), deepcopy(bipe_sac_create_config)]
+    exp_path = '/mnt/nfs/wanghaolin/cont_bc/DI-engine/pendulum_td3_seed0/ckpt/ckpt_best.pth.tar'
+    expert_policy = load_policy(train_config, load_path=exp_path, seed=0)
+
+    # collect expert demo data
+    collect_count = 400
+    expert_data_path = 'expert_data_dqn.pkl'
+    state_dict = expert_policy.collect_mode.state_dict()
+    collect_config = [deepcopy(bipe_sac_config), deepcopy(bipe_sac_create_config)]
+    # collect_demo_data(
+    #     collect_config, seed=0, state_dict=state_dict, expert_data_path=expert_data_path, collect_count=collect_count
+    # )
+    collect_episodic_demo_data(
+        deepcopy(collect_config), seed=0, state_dict=state_dict, expert_data_path=expert_data_path, collect_count=collect_count
+    )
+    collect_episodic_demo_data(
+        collect_config, seed=0, state_dict=state_dict, expert_data_path='eval_data.pkl', collect_count=10,
+    )
```

Comment:
Why do you collect twice?

Reply:
the second time of collecting is used for validation